common/app/implicits/ItemKickerImplicits.scala

```diff
@@ -56,11 +56,11 @@ object ItemKickerImplicits {
       case AnalysisKicker => kickerHtml
       case ReviewKicker => kickerHtml
       case CartoonKicker => kickerHtml
-      case PodcastKicker(_, series) => kickerHtml
-      case TagKicker(_, name, url, id) => kickerHtml
-      case SectionKicker(_, name, url) => kickerHtml
-      case FreeHtmlKicker(_, body) => kickerHtml
-      case FreeHtmlKickerWithLink(_, body, url) => kickerHtml
+      case PodcastKicker(_, _) => kickerHtml
+      case TagKicker(_, _, _, _) => kickerHtml
```

Comment:
I suspect this is all automated (if so ignore) but if we're never binding parameters, the `_: TagKicker` match approach is probably clearer.